File: evidence/generated-systems/angular/governance-on-angular/src/app/models/Organization.js

// Define collection and schema for Organization
export interface Organization {
    name:
	type : string
    legalName:
	type : string
    jurisdiction:
	type : string
    industrySector:
	type : string
    GovernanceBodies:
 	type : [{ type: Schema.Types.ObjectId, ref: 'GovernanceBody' }]
    Policies:
 	type : [{ type: Schema.Types.ObjectId, ref: 'Policy' }]
    Risks:
 	type : [{ type: Schema.Types.ObjectId, ref: 'Risk' }]
    ThirdParties:
 	type : [{ type: Schema.Types.ObjectId, ref: 'ThirdParty' }]
    RecordsRepositories:
 	type : [{ type: Schema.Types.ObjectId, ref: 'RecordsRepository' }]
    DataProcessingActivities:
 	type : [{ type: Schema.Types.ObjectId, ref: 'DataProcessingActivity' }]
    CompliancePrograms:
 	type : [{ type: Schema.Types.ObjectId, ref: 'ComplianceProgram' }]
    AuditPrograms:
 	type : [{ type: Schema.Types.ObjectId, ref: 'AuditProgram' }]
    BusinessUnits:
 	type : [{ type: Schema.Types.ObjectId, ref: 'BusinessUnit' }]
    Matters:
 	type : [{ type: Schema.Types.ObjectId, ref: 'Matter' }]
    DataBreaches:
 	type : [{ type: Schema.Types.ObjectId, ref: 'DataBreach' }]
#
    collection: 'organizations'
}

File: evidence/generated-systems/rubyonrails/governance-on-rails/governanceonrails/app/models/organization.rb
class Organization < ApplicationRecord


  has_many :GovernanceBodies, class_name: 'GovernanceBody'
  has_many :Policies, class_name: 'Policy'
  has_many :Risks, class_name: 'Risk'
  has_many :ThirdParties, class_name: 'ThirdParty'
  has_many :RecordsRepositories, class_name: 'RecordsRepository'
  has_many :DataProcessingActivities, class_name: 'DataProcessingActivity'
  has_many :CompliancePrograms, class_name: 'ComplianceProgram'
  has_many :AuditPrograms, class_name: 'AuditProgram'
  has_many :BusinessUnits, class_name: 'BusinessUnit'
  has_many :Matters, class_name: 'Matter'
  has_many :DataBreaches, class_name: 'DataBreach'

end
